src/auspex/filters/correlator.py

```diff
@@ -19,17 +19,8 @@ class Correlator(ElementwiseFilter):
     def operation(self):
         return np.multiply
 
-    def update_descriptors(self):
-        logger.debug('Updating correlator "%s" descriptors based on input descriptor: %s.', self.name, self.sink.descriptor)
+    def filter_name(self):
```

Comment:
What is this `filter_name` method for? It seems only partially implemented, since you didn't define it for most of of the other filter classes.